compiler/rustc_ty_utils/src/layout.rs

```diff
@@ -561,6 +563,22 @@ fn layout_of_uncached<'tcx>(
 
             let e_ly = cx.layout_of(e_ty)?;
 
+            // Check for the rustc_simd_monomorphize_lane_limit attribute and check the lane limit
+            if let Some(limit) = find_attr!(
```

Comment:
I wouldn't call it very likely, but since we run this for every ADT this could be a perf regression. I say unlikely because getting attrs is not super expensive but we don't call `get_all_attrs` inside this query at the moment.

Reply:
oh you're right, alright then this shouldn't be much of an issue